package/libretro-cores/0002-fix-rules.d.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/rules.d/core-rules.sh b/rules.d/core-rules.sh
+index f80055be..a355d27b 100644
+--- a/rules.d/core-rules.sh
++++ b/rules.d/core-rules.sh
+@@ -1121,7 +1121,7 @@ include_core_picodrive() {
+ libretro_picodrive_name="Picodrive"
+ libretro_picodrive_git_url="https://github.com/libretro/picodrive.git"
+ libretro_picodrive_git_submodules="yes"
+-libretro_picodrive_build_makefile="Makefile.libretro"
++libretro_picodrive_build_makefile="Makefile.libretro -f ../Makefile.libretro-picodrive"
```

Comment:
Build fails the normal way? Maybe we should alert libretro folks.

Reply:
For miyoo platform many cores have hardcoded path In var CC=/opt/miyoo/usr/bin/arm-linux-gcc

Most of them I can override using env, but in case of picodrive is tricky, because inside during compilation they reassign CC to host-gcc.

The best option for us is to have extra env with path to gcc like for other platforms e.g.
CC=$EXTRA_PATH/arm-linux-gcc